src/shared/vscode/env.ts

```diff
@@ -95,3 +95,9 @@ export function getMinVscodeVersion(): string {
 export function getMinNodejsVersion(): string {
     return packageJson.devDependencies['@types/node'].replace(/[^~]/, '')
 }
+
+export function userAgent() {
```

Comment:
This would use `getClientId` from https://github.com/aws/aws-toolkit-vscode/pull/2680 once merged